libs/domain/cart/services/src/adapter/default-cart.adapter.ts

```diff
@@ -203,27 +203,32 @@ export class DefaultCartAdapter implements CartAdapter {
   }
 
   create(qualifier?: CreateCartQualifier): Observable<Cart> {
-    return combineLatest([
-      this.store.get(),
-      this.currency.get(),
-      this.priceMode.get(),
-    ]).pipe(
+    const request = (attributes: CreateCartQualifier) =>
+      this.http.post<ApiCartModel.Response>(
+        `${this.SCOS_BASE_URL}/${ApiCartModel.UrlParts.Carts}`,
```

Comment:
I wonder can we start using endpoint service here cc @dunqan

Reply:
we don't have endpoints services, but we have generic handling of includes